NUnitLite/TouchRunner/TouchRunner.cs

```diff
@@ -140,14 +140,22 @@ public void AutoRun ()
 			});
 		}
 
+		bool running;
 		public void Run ()
 		{
+			if (running) {
+				Console.WriteLine ("Not running because another test run is already in progress.");
+				return;
+			}
+
 			if (!OpenWriter ("Run Everything"))
 				return;
 			try {
+				running = true;
```

Comment:
This happens when running our watchOS tests an device, somehow the tests are autorun twice in the same process.

In theory it should be impossible, because the autorun request is done using environment variables, which are read at launch (so there should never be two autorun requests in the same process).

Yet it sometimes happens when the device automatically restarts the watch extension (after it has crashed for instance), and then I try to run the tests again from the command line.

Since it's random, and tracing down problems on the watch device is slow, I came up with this simple patch which seems to work.